va_explorer/va_analytics/dash_apps/va_dashboard.py

```diff
@@ -812,34 +884,118 @@ def shift_granularity(current_granularity, levels, move_up=False):
 
 # =========Map Metrics =======================#
 # Top metrics to track for map dropdown
-@app.callback(
+# callback 5
```

Comment:
delete comment chunk if not helpful, especially if its code you commented out. If they're helpful comments keep them with good judgement